docs/tutorials/basics.ipynb

```diff
@@ -68,7 +68,7 @@
     "id": "67gUU740MSXd"
```

Comment:
<p>"These Device can be" fails grammar check.</p><p> </p><p>We are getting rid of devices in circuit next version, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Yes, I've removed this language.</p>